pages/common/fping.md

```diff
@@ -0,0 +1,15 @@
+# fping
+
+> A more powerful ping which ping multiple hosts
+
+- List alive host within subnet via netmask
```

Comment:
"host" --> "hosts"; I'd also suggest "via" --> "using a", which I think is clearer. By the way, does the `-g` stand for any word in particular?